doc/scopes.rst

```diff
@@ -0,0 +1,30 @@
+Dune projects and workspaces
+============================
+
+Whenever Dune builds anything, it does so at the level of a *Dune workspace*. A
+Dune workspace is a set of Dune projects. A typical workspace consists of a
+single Dune project.
+
+A Dune project is defined by the presence of a
+:doc:`/reference/dune-project/index` file. Each Dune project extends over the
+subtree rooted at the directory containing the
+:doc:`/reference/dune-project/index` file, excluding any nested Dune projects.
+
+The root of the current workspace is determined by the outermost
+:doc:`/reference/dune-project/index` file in an ancestor of the current
+directory or by the presence of a :doc:`/reference/dune-workspace/index` file
+(see :ref:`finding-root` and :ref:`forcing-root` for details). All
+:doc:`/reference/dune-workspace/index` files other than the one specifying the
+root of the workspace are ignored.
```

Comment:
The last sentence is about `dune-workspace`, it could go to the dedicated paragraph currently on lines 24-25.

Reply:
Fixed, thanks.